README.md

```diff
@@ -14,6 +14,21 @@ The project was *initially* forked from the  upstream [mozilla-services/go-cose]
 
 We thank the [Mozilla maintainers and contributors][mozilla-contributors] for their great work that formed the base of the [veraison/go-cose][veraison-go-cose] project.
 
+## Community
+
+The [veraison/go-cose](https://github.com/veraison/go-cose) project is an open source community effort.
+
+You can reach the go-cose community via::
+
+- [Mailing List](https://groups.io/g/go-cose-community)
```

Comment:
We already have a Mailing List known as:
[veraison-project@confidentialcomputing.io](mailto:veraison-project@confidentialcomputing.io)

We should continue with this one ?
We can discuss this in our coming meeting.

Reply:
We haven't had much activity for a mailing list and have mostly been responsive to PRs and Issues. We can discuss in the meeting for whether we should continue under the confidentialcomputing.io or use the groups.io.